cloud/scope/clients.go

```diff
@@ -26,27 +26,30 @@ import (
 	"github.com/oracle/cluster-api-provider-oci/cloud/services/computemanagement"
 	containerEngineClient "github.com/oracle/cluster-api-provider-oci/cloud/services/containerengine"
 	identityClient "github.com/oracle/cluster-api-provider-oci/cloud/services/identity"
+	lbs "github.com/oracle/cluster-api-provider-oci/cloud/services/loadbalancerservice"
 	nlb "github.com/oracle/cluster-api-provider-oci/cloud/services/networkloadbalancer"
 	"github.com/oracle/cluster-api-provider-oci/cloud/services/vcn"
 	"github.com/oracle/cluster-api-provider-oci/version"
 	"github.com/oracle/oci-go-sdk/v65/common"
 	"github.com/oracle/oci-go-sdk/v65/containerengine"
 	"github.com/oracle/oci-go-sdk/v65/core"
 	"github.com/oracle/oci-go-sdk/v65/identity"
+	"github.com/oracle/oci-go-sdk/v65/loadbalancer"
 	"github.com/oracle/oci-go-sdk/v65/networkloadbalancer"
 	"github.com/pkg/errors"
 	"k8s.io/klog/v2/klogr"
 )
 
 // OCIClients is the struct of all the needed OCI clients
 type OCIClients struct {
-	ComputeClient           compute.ComputeClient
-	ComputeManagementClient computemanagement.Client
-	VCNClient               vcn.Client
-	LoadBalancerClient      nlb.NetworkLoadBalancerClient
-	IdentityClient          identityClient.Client
-	ContainerEngineClient   containerEngineClient.Client
-	BaseClient              base.BaseClient
+	ComputeClient             compute.ComputeClient
+	ComputeManagementClient   computemanagement.Client
+	VCNClient                 vcn.Client
+	LoadBalancerClient        nlb.NetworkLoadBalancerClient
```

Comment:
please rename I think we should call this LBaaS loadbalancer type? Or just Load balancer type, thats what console does, there are 2 types in console, Load Balancer and Network Load Balancer. Can you please rename the nlb related files start to nlb please? -> NetworkLoadBalancerClient and rename the new client to LaodBalancerClient

Reply:
Yes. I'll do that.